Added some math content (#14)
0xPolygon/devrel-docs: docs/zk/np_completeness.md

# NP-Completeness

NP-completeness is a concept from computational complexity theory, which is a
field that studies the difficulty of solving different computational problems.

> To explain it in a way that avoids heavy math jargon but is still rigorous,

Reviewer: ```suggestion
To explain it in a way that avoids heavy math jargon, though still rigorous,
```
